actionpack/lib/action_dispatch/routing/route_set.rb

```diff
@@ -579,6 +580,8 @@ def url_options; {}; end
             extend path_helpers
           end
 
+          extend AbstractController::IgnoreActionMethods
```

Comment:
`_routes` is defined in that module, which is included directly in the controller. It's a public method but we don't want to include it in `action_methods`.